Terminal.Gui/Views/TabView.cs

```diff
@@ -19,8 +18,8 @@ public class TabView : View
     /// <summary>This sub view is the 2 or 3 line control that represents the actual tabs themselves.</summary>
     private readonly TabRowView _tabsBar;
 
-    private Tab _selectedTab;
-    private TabToRender [] _tabLocations;
+    private Tab? _selectedTab;
+    private TabToRender []? _tabLocations;
```

Comment:
`TabView` is rendering badly again during the tab select changing.

Reply:
It seems you merged an old version of `TabView`, before the changes I made,

Reply:
Please ignore Tabview for now.